lib/task_pulse.js

```diff
@@ -15,7 +15,8 @@ TaskPulse.prototype = {
     debug('creating taskcluster pulse listener');
     var listener = new PulseListener({credentials: {
       username: this.config.taskPulseConfig.username,
-      password: this.config.taskPulseConfig.password
+      password: this.config.taskPulseConfig.password,
+      queueName:  this.config.taskPulseConfig.queueName
     }});
```

Comment:
Oh, should probably be:
```js
var listener = new PulseListener({
  queueName:  this.config.taskPulseConfig.queueName,
  credentials: {
    username: this.config.taskPulseConfig.username,
    password: this.config.taskPulseConfig.password
  }
});
```

Ie. `queueName` is not part of the `credentials`.dictionary. Other than that this looks good.